spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -489,13 +486,9 @@ public SparkContext createSparkContext_1() {
     if (execUri != null) {
       conf.set("spark.executor.uri", execUri);
     }
-    if (System.getenv("SPARK_HOME") != null) {
-      conf.setSparkHome(System.getenv("SPARK_HOME"));
```

Comment:
there is still the "included version of spark" in Zeppelin? In that case SPARK_HOME is not set?
http://zeppelin.apache.org/docs/0.7.2/interpreter/spark.html#2-set-master-in-interpreter-menu